sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClientBuilder.java

```diff
@@ -280,7 +280,19 @@ private String modifyUserAgentString(String applicationId, Configuration userAge
         String userAgent = UserAgentUtil.toUserAgentString(applicationId, BLOB_CLIENT_NAME, BLOB_CLIENT_VERSION,
             userAgentConfiguration);
         Matcher matcher = pattern.matcher(userAgent);
-        String version = encryptionVersion == EncryptionVersion.V2 ? "2.0" : "1.0";
+        String version;
+        switch (encryptionVersion) {
```

Comment:
Any thoughts on a cleaner way to handle mapping the version enums to strings? Or are case statements usually the way to go here?